src/spot_skills/scripts/arm_long_trajectory_demo.py

```diff
@@ -82,54 +80,35 @@ def main():
     # By now, Spot should be powered on and controllable
     arm_controller.deploy_arm()
 
-    # Define the reference start time for the initial trajectory
-    start_time_s = time.time()
-    ref_timestamp = TimeStamp.from_time_s(start_time_s)
-
-    # To ensure the trajectory runs smoothly, move to its initial position and velocity
-    point_0 = query_trajectory(0)
-    point_0.time_from_start_s = TRAJ_APPROACH_TIME_S  # Take some time to reach start
-
-    joint_names = ["sh0", "sh1", "el0", "el1", "wr0", "wr1"]
-    go_to_point_0 = JointTrajectory(joint_names, ref_timestamp, [point_0])
+    ### Compute the full trajectory to be executed on Spot's arm ###
+    dt_s = 0.2  # Timestep (seconds)
+    relative_t_s = 0  # Relative time (seconds) since trajectory started
```

Comment:
Should the first point have its timestep `dt_s` after the reference?

Reply:
I don't think so - The example code uses `query_arm_joint_trajectory` on `segment_start_time + i * dt`, which should be zero for the first point.